lib/streamlit/elements/markdown.py

```diff
@@ -17,6 +17,7 @@
 from streamlit import type_util
 from streamlit.proto.Markdown_pb2 import Markdown as MarkdownProto
 from .utils import clean_text
+from ..type_util import SupportsStr
```

Comment:
nit: would slightly prefer to import from `streamlit.type_util` so that relative imports are only used for imports in the same directory.

(I don't think we have any strict convention for this in the codebase, but the code does for the most part follow it as far as I know)

Reply:
Fixed.